http/src/main/resources/init-resources/startup.sh

```diff
@@ -68,6 +67,7 @@ export START_USER_SCRIPT_URI=$(startUserScriptUri)
 export START_USER_SCRIPT_OUTPUT_URI=$(startUserScriptOutputUri)
 export WELDER_MEM_LIMIT=$(welderMemLimit)
 export MEM_LIMIT=$(memLimit)
+export INIT_BUCKET_NAME=$(initBucketName)
 export USE_GCE_STARTUP_SCRIPT=$(useGceStartupScript)
 GPU_ENABLED=$(gpuEnabled)
 export IS_RSTUDIO_RUNTIME="false" # TODO: update to commented out code once we release Rmd file syncing
```

Comment:
Not related to this ticket, but is this TODO still valid?

RMD file syncing is released currently as far as I'm aware. cc @gpcarr

Reply:
I think it's turned off for now. Once @gpcarr 's current rmd ticket is done, we should be able to turn it back on

Reply:
yeah it's still valid - we're hardcoding to `false` until everything is in place for Rmd file syncing. I'm not sure we'd be able to turn it on when my backend changes are done though, I think we'd need to wait for the frontend to be ready as well